PYME/LMVis/views.py

```diff
@@ -153,6 +153,14 @@ def __div__(self, scalar):
                     self.clip_plane_position / scalar
                     )
 
+    def __floordiv__(self, scalar):

```

Comment:
what exactly is the bug this is trying to fix??? In any case I think you've got the fix wrong. Everything here should be float, including scalar, so there is no way you should be casting scalar to an int. The only potential change I could see (which is more about ensuring correct behaviour on py2 than fixing anything on py3) would be to cast scalar to a float in `__div__`.

Reply:
I see, `__div__` doesn't exist on py3. For the View object we have no need to define `__floordiv__` (and so should leave it undefined). `__truediv__ should be identical to `__div__` - ie just a line under the definition of `__div__` saying `__truediv__ = __div__`. Could potentially also add a cast to float in `__div__` to make things safer on py2 .

Reply:
I added `__floordiv__` just in case. I think it is supposed to be integer division. If you look in `animation.py` on line 196, it will default to integer division in py2. Therefore I think we do want to cast to int, not float.